### ISSUE TITLE
VPN backend starts fine, but GUI doesn't launch (macOS 15.X)

### ISSUE THREAD

**Issue by nemorosus** [mullvad/mullvadvpn-app#7078]
### Is it a bug?

- [X] I know this is an issue with the app, and contacting Mullvad support is not relevant.

### I have checked if others have reported this already

- [X] I have checked the issue tracker to see if others have reported similar issues.

### Current Behavior

So, I wrote on email but I guess this is the better solution so all devs and everyone can see, maybe someone else had this issue and knows how to fix it, I don't know.

The issue I have is I cannot get the lock-bar on the top to launch itself, I can launch it only manually. I done almost every trick in the book I know, adding it from System Preferences manually doesn't work, the Dock login on startup doesn't work, and inside the GUI as well the option for the auto-startup doesn't work.

I am running on the latest .7 beta, just installed it now to see if it'll fix the issue, however the issue is still present.

![image](https://github.com/user-attachments/assets/e3a75fc0-0a43-429f-89dd-4966dd52da7c)

Mullvad indeed connects in the backend, however for the GUI to appear, I must do it manually. Any suggestions, this might be a system-specific issue with my MacOS?

Otherwise everything is okay. When I open the app from the dock manually...
![image](https://github.com/user-attachments/assets/cdf80201-96bf-4545-bcc7-b1903cc6c028)

### Expected Behavior

N/A

### Steps to Reproduce

1. Auto-startup doesn't work on 15.0.1

### Failure Logs

```shell
Too long to post directly, here is a self-hosted link:
https://pub-1ed977f1def942878eb45e778a6e653f.r2.dev/mullvad-report.log
```

### Operating system version

MacOS 15.0.1

### Mullvad VPN app version

2024.7β

**Comment by nemorosus** [comment]
Nobody with the same issue as me? :((((((

**Comment by nemorosus** [comment]
It's not in the section "Open at login" however it is present in "Allow in background" in the Preferences/Settings.

This is the log output:
[2024-10-27 20:40:28.780][verbose] Chromium sandbox is enabled
[2024-10-27 20:40:28.784][info] Running version 2024.7-beta1
[2024-10-27 20:40:28.940][info] Detected locale: en-US
[2024-10-27 20:40:28.960][info] Connected to the daemon
[2024-10-27 20:40:29.041][info] Autoconnect the tunnel
[2024-10-27 21:40:28.955][verbose] GRPC Channel connectivity state changed to 2

However this could very well be when I launched it manually from the apps. So, the VPN actually starts in the background without any issues, I've tested it on Mullvads website - it autoconnects, however the GUI icon in the menubar is not present until I manually open it myself.
